test/remote/gateways/remote_merchant_e_solutions_test.rb

```diff
@@ -191,9 +191,11 @@ def test_connection_failure_404_notfound_with_purchase
   end
 
   def test_successful_purchase_with_3dsecure_params
-    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(
-      { :xid => 'ERERERERERERERERERERERERERE=',
-        :cavv => 'ERERERERERERERERERERERERERE='}))
+    options = @options.merge(
+      { xid: 'ERERERERERERERERERERERERERE=',
+        cavv: 'ERERERERERERERERERERERERERE='}
+    )
+    assert response = @gateway.purchase(@amount, @credit_card, options)
```

Comment:
18 tests, 77 assertions, 1 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
94.4444% passed

Same results on my branch and on master. The failing test is returning a `401` error instead of a `404` error: `test_connection_failure_404_notfound_with_purchase`.